.travis.yml

```diff
@@ -33,6 +33,24 @@ jobs:
       install: pip install moban
       script: .ci/check_moban.sh
 
+    - stage: lint
+      language: python
+      python: 3.6
+      cache: pip
+      before_install:
+        - export PATH=$PATH:./node_modules/.bin
+        - export NODE_PATH=$NODE_PATH:./node_modules
+      install:
+        - pip install coala-bears
+        - npm install
+      script: coala --non-interactive -V
+      install: false
+      before_script: false
+      script: false
+      after_success: false
+      before_deploy: false
+      deploy: false
```

Comment:
use `<<: *disable_global` here, like the example you have been given.

Otherwise the lint job is inaccurately marked as "Node.js: 6 Python 3.6" on the matrix
https://travis-ci.org/coala/git-task-list/builds/391029250

Reply:
*disable_global Doesn't work, There is a failed build.

Reply:
https://travis-ci.org/coala/git-task-list/jobs/391031526

Reply:
I needs before_script global hook.

Reply:
That log is for the commit
https://github.com/coala/git-task-list/commit/d421a130aa21d2a00d47385ec82c948aa51bee40#diff-354f30a63fb0907d4ad57269548329e3R36
which doesnt use disable_global. And i put a comment on that commit showing why it failed.